core/storage/user/gae_models.py

```diff
@@ -132,6 +132,27 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE_AT_END
 
+    @staticmethod
+    def get_model_association_to_user():
+        """Model is exported as a one instance per user."""
+        return base_models.MODEL_ASSOCIATION_TO_USER.ONE_INSTANCE_PER_USER
+
+    @staticmethod
+    def get_field_names_for_takeout():
+        """Export renames some time related fields to clearly indicate that
```

Comment:
The export method renames ...

Also, nit: time-related

Reply:
done